Bottom: 731ecb02c32b2cfd0470c647256283b6e18f396e
Top:    25357fdb2252d11336d7583d5fcccf46c77a3cef
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-04-10 16:31:02 -0400

Refresh of jds-audio

---

diff --git a/sound/soc/fsl/Kconfig b/sound/soc/fsl/Kconfig
index 3dcc068..d480a73 100644
--- a/sound/soc/fsl/Kconfig
+++ b/sound/soc/fsl/Kconfig
@@ -54,7 +54,7 @@ config SND_MPC52xx_SOC_EFIKA
 
 config SND_MPC52xx_SOC_DSPEAK01
 	tristate "SoC I2S Audio support for Digispeaker DSPEAK01 with TAS5504"
-	depends on SND_MPC52xx_SOC && PPC_DSPEAK01
+	depends on PPC_DSPEAK01
 	select SND_SOC_MPC5200_I2S
 	select SND_SOC_TAS5504
 	help
